README.md

```diff
@@ -168,6 +177,13 @@ note that the `cmd` argument **must be quoted**.
 
 The `cmd` and `dir` arguments can also expand the same special keywords as `:h expand` e.g.
 `TermExec cmd="echo %"` will be expanded to `TermExec cmd="echo /file/example"`
+
+The `size` and `direction` arguments are like the `size` and `direction` arguments of `ToggleTerm`.
+
+By default focus is returned to the original window after executing the command
+(except for floating terminals).
+Use argument `go_back=0` or `go_back=false` to disable this behaviour.
```

Comment:
Sorry I should have been clearer, I think go back should be one type of value or the other. I think it taking both a number and a false or true string is too many permutations. I just meant it should be true or false, no numbers.

Reply:
But `go_back=0` is easier to type.

Reply:
@zeertzjq tbh I don't imagine people will be manually typing out `go_back` in the cli often and are more likely to map it so something. Tbh I hate the vim convention of 0 as falsy and 1 or greater as true. I'd much rather this was just clear than necessarily just easy to type.

Another potential approach would be having a flag value something like `--focus-term` or something so just the presence of the flag is taken as truthy.

Reply:
Maybe it can be changed to just a string `focus` instead.

Reply:
Regarding brevity, one day I hope to add commandline completion for the plugin so tbh I don't think the length of the keys will be that big a deal once a user can tab complete but definitely don't want this to have 2 different options.

I don't think a naked string argument is a good idea either I want the option to the commandline arguments to be `key=value` so parsing can be consistent. I really don't think the 3-4 characters saved is worth it here for an option that I feel confident that the vast majority of users will not be typing since it wasn't even available beforehand. I don't mind it being exposed but not at the cost of using numbers as booleans.

Reply:
You know what, it's fine, you can just change it to `go_back=0|1` I personally hate this pattern so might still change it in the future but it's not that big a deal.